server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/Capability.scala

```diff
@@ -24,9 +24,7 @@ import java.net.URI
 import org.apache.james.jmap.model.CapabilityIdentifier.{JMAP_CORE, JMAP_MAIL}
 
 final case class CapabilityIdentifier(value: URI) {
-  def asString(): String = {
-    value.toString
-  }
+  def asString(): String = value.toString
```

Comment:
I think you don't even need the `()` either

`def asString: String = value.toString`

Reply:
what about `val asString = value.toString` ?

Reply:
and we can close this kind conversation forever